src/main/java/com/spring/studentmanagement/models/Role.java

```diff
@@ -23,4 +25,14 @@ public class Role {
 
     @Column(name = "role_name", length = 45, unique = true, nullable = false)
     private String roleName;
+    @OneToMany(mappedBy = "role")
+    private Collection<AppUser> appUser;
+
+    public Collection<AppUser> getAppUser() {
+        return appUser;
```

Comment:
To be removed
Why this? This is a Bi directional relation which we don't need.
The getAppUser and setAppUser methods are not used in the program.
YAGNI principle.